core/net/ipv6/sicslowpan.c

```diff
@@ -169,6 +169,14 @@ void uip_log(char *msg);
 #define COMPRESSION_THRESHOLD 0
 #endif
 
+/** \brief Fixed size of a frame header. This value is
+ * used in case framer returns an error or if SICSLOWPAN_USE_FIXED_HDRLEN
+ * is defined.
+ */
+#ifndef SICSLOWPAN_FIXED_HDRLEN
+#define SICSLOWPAN_FIXED_HDRLEN 21
```

Comment:
This is a minor change that really doesn't affect `core`, but normally one we would prefer in a different PR, but as it is already in a separate commit I don't object :+1: